ui/app/rif/components/modal/addNewTokenNetworkAddress/addNewChainAddressToResolver.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
```

Comment:
I don't get it, is this deleted or not? down below says that you deleted this file, anyway i think this file is not needed, i mean you should use the generic modal instead of adding new modal templates, that's why we have the elements logic and everything to use that. Can you explain why you have this?

Reply:
This was moved to the folder of Domain Active in next commit.